sdk/keyvault/keyvault-admin/src/accessControlClient.ts

```diff
@@ -436,13 +436,14 @@ export class KeyVaultAccessControlClient {
   }
 
   /**
-   * Upserts a role definition in an Azure Key Vault.
+   * Sets a role definition in an Azure Key Vault.
```

Comment:
Hmm I think it’ll be good to say that this creates or updates. I like `set` in the name, but the docs could be a bit clearer, just in case someone finds it confusing.